dxr/plugins/clang/filters.py

```diff
@@ -100,9 +115,9 @@ class CallerFilter(_CQualifiedNameFilter):
     name = 'callers'
     description = Markup('Functions which call the given function or method: <code>callers:GetStringFromName</code>')
 
-    def __init__(self, term):
+    def __init__(self, term, enabled_plugins):
```

Comment:
We call this just `plugins` down in FilterAggregator. Let's choose one or the other.